src/main/webapp/my_advertisements.xhtml

```diff
@@ -0,0 +1,174 @@
+<ui:composition xmlns="http://www.w3.org/1999/xhtml"
+                xmlns:ui="http://java.sun.com/jsf/facelets"
+                xmlns:p="http://primefaces.org/ui"
+                xmlns:f="http://java.sun.com/jsf/core" xmlns:h="http://java.sun.com/jsf/html"
+                template="/templates/page_view.xhtml">
+
+  <ui:define name="general_content">
+    <h:panelGroup styleClass="layout-content">
+
+      <h:form id="form">
+        <div class="card crud-demo">
```

Comment:
There's going to be a problem if multiple div's have the same class and if we decide to write some css for div elements.

Reply:
Why you think there will be a problem? Css classes are meant to be used to style multiple elements the same way, as far as I know.

Reply:
Oh yeah, I forgot how that works. 😅